### PR TITLE
Basic page structure: Implement Basic Page Structure of Leaderboard app

### PR DESCRIPTION
**Pull request by ClaudiaRojasSoto**
In this pull request, I'm proposing the implementation of the basic page structure for my Leaderboard list app. This initial version focuses on functionality and follows the provided wireframe, with minimal styling applied. The goal is to set up the JavaScript project using webpack and ES6 modules while adhering to best practices and Gitflow.

## General Updates:

- [x] Resolve all linter errors and ensured clean code.
- [x] Follow Gitflow workflow for branch management and feature development.
- [x] Maintain professional documentation throughout the project.

## Project Updates:

- [x] Create an npm project with webpack, setting up the build process and bundling.
- [x] Develop the basic HTML markup following the wireframe provided. Minimal styling has been applied, focusing on the layout structure.

## Note:
Please review the changes and provide feedback on the implemented basic page structure. Keep in mind that this pull request focuses on functionality rather than style. I have ensured no linting errors, followed the Gitflow workflow, and maintained professional documentation. Your feedback and suggestions for improvement are greatly appreciated.

Greetings,
@ClaudiaRojasSoto

### CODE REVIEW

File: Readme.md [hunk 1]
```diff
@@ -0,0 +1,176 @@
...
+
+> -   GitHub Docs for providing a wealth of information on Git and GitHub.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT licensed](https://github.com/ClaudiaRojasSoto/Leaderboard/blob/master/MIT.md).
```
Comment (rustam-tolipov) [inline Readme.md]: - [x] If I want to see your LICENSE, it shows a broken link `404` kindy fix it and your project will be ready to approve.

**Review by rustam-tolipov — CHANGES_REQUESTED** [review]
Hello @ClaudiaRojasSoto,

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
_Your project is almost complete!_
_No Linters errors_
_Great setup_

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Check the comments under the review._

You can also consider:
N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: Readme.md (hunk 1 above)
Comment (ClaudiaRojasSoto) [reply Readme.md:174]: @Rustamxon7 I tried to fix that but I could not, can you give me some light on how to implement this link in a good way, please?

**Review by Shedrack-Sunday — APPROVED** [review]
Hi @ClaudiaRojasSoto        👋

Congratulations! 🎉 Your project is complete and ready to be merged! 🚀

image


## Highlights
- Good job with the use of GitHub features✔️ 
- Kudos on implementing the feature structure as requested ✔️ 

## Optional Suggestions 💡
While these comments with the [OPTIONAL] prefix won't prevent the approval of this pull request, I strongly recommend taking them into account as they can improve your code. Some of them were missed by the previous reviewer and addressing them will really enhance your application.
Cheers, and Happy coding!👏👏👏

If you have any questions or need clarification on anything, feel free to leave a comment in the pull request thread, and don't forget to tag me so I'll receive the notification.

As per the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561), you have a limited number of reviews per project. If you feel that the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: Readme.md (hunk 1 above)
Comment (rustam-tolipov) [reply Readme.md]: I suggest you to copy and past this link:

```
https://github.com/ClaudiaRojasSoto/Leaderboard/blob/basic-page-structure/MIT.md

**Pull request merged**